tests/integration/basic/test_regression.py

```diff
@@ -41,11 +41,15 @@ def test_0_predict_file_flow(self):
 
         predictions = predictor.predict(df)
 
+        tests = predictor.test(df, [r2_score])
+
         # sanity checks
         self.assertTrue(r2_score([float(x.rstrip('$')) for x in df[target]], predictions['prediction']) > 0.8)
         self.assertTrue(all([0 <= p <= 1 for p in predictions['confidence']]))
         self.assertTrue(all([p['lower'] <= p['prediction'] <= p['upper'] for _, p in predictions.iterrows()]))
 
+        self.assertTrue(tests['r2_score'].dtypes == 'float64')
```

Comment:
This check doesn't seem correct. `r2_score` would be the value in the `Metric` column, not the actual name of the column.

Reply:
```suggestion
        self.assertTrue(tests['Value'].dtypes == 'float64')
```